src/tree/v2/application/ApplicationResourceBranchDataProviderManager.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ApplicationResource, BranchDataProvider, ResourceModelBase } from "../../../api/v2/v2AzureResourcesApi";
+import { ResourceBranchDataProviderManagerBase } from '../ResourceBranchDataProviderManagerBase';
+
+export class ApplicationResourceBranchDataProviderManager extends ResourceBranchDataProviderManagerBase<BranchDataProvider<ApplicationResource, ResourceModelBase>>{
+    constructor(
+        defaultProvider: BranchDataProvider<ApplicationResource, ResourceModelBase>,
+        extensionActivator: (type: string) => void
+    ) {
+        super(
+            defaultProvider,
+            extensionActivator);
```

Comment:
Nit
```suggestion
            extensionActivator
        );
```

Reply:
We should add some ESLint rules for this.  Updated.